### DISCUSSION TITLE
Unable to load...

### DISCUSSION THREAD

**Discussion by C00K50N** [nathanmarlor/foxess_modbus_charge_period_card#19] [Support] [answered]
Hmm... not sure what I've missed here.  Just installed this card, after switching from direct LAN connection to now an RS485 to Ethernet device and migrating to the FoxESS - Modbus integration.  I'd like to be able to set charge times via HA, but I'm getting this error:
![image](https://github.com/nathanmarlor/foxess_modbus_charge_period_card/assets/102669305/f0095b84-427a-4117-8fbe-91e65a4346f4)

As far as I can tell they're enabled, but I must be missing something...
![image](https://github.com/nathanmarlor/foxess_modbus_charge_period_card/assets/102669305/10d5412a-cf49-42d0-9f81-d4f5d9761861)

Can anyone help?

**Comment by canton7** [comment]
Can you check the entity ids of those entities?

**Reply by C00K50N** [reply]
![image](https://github.com/nathanmarlor/foxess_modbus_charge_period_card/assets/102669305/681aba66-1780-4383-bc6d-40667a367efc)
That's one of them...

**Reply by canton7** [reply, marked as answer]
Hmm, looks fine. 

The error means that those entites aren't in the list of entites which HA is giving to the UI cards. Try restarting HA, see if that helps? Also check the logs after restart, see if there are any errors setting up the FoxESS - Modbus integration.

**Reply by canton7** [reply]
Related, perhaps: https://github.com/nathanmarlor/foxess_modbus/issues/430

**Reply by C00K50N** [reply]
Yep, a restart sorted it.  Sorry for wasting your time - should have tried that first.  Thanks so much for your help.

**Reply by canton7** [reply]
Glad that worked! I'll put something in the error message about that.
